### ISSUE TITLE
Error setting up TOP as a dependency in 1.14 dev environment

### ISSUE THREAD

**Issue by pupnewfster** [McJty/TheOneProbe#344]
I am having an issue adding TOP 1.14 (1.14-1.4.37-12) to my dev environment if I try to make it be deobf'd.

The below error occurs when I have
```
compileOnly fg.deobf("mcjty.theoneprobe:TheOneProbe-1.14:${top_version}:api")
runtimeOnly fg.deobf("mcjty.theoneprobe:TheOneProbe-1.14:${top_version}")
```
Caused by: org.gradle.internal.resolve.ModuleVersionNotFoundException: Could not find net.minecraftforge:forge:1.14.4-28.1.44_mapped_snapshot_20191011-1.14.3_at_f0945e3cd9cfc96a1d7462229a035f78dab1da5d.

But if I remove the `fg.deobf` it is able to resolve it (though it doesn't work due to being obfuscated when the rest of the dev environment is not.) Looking at the pom file on the [maven](https://maven.tterrag.com/mcjty/theoneprobe/TheOneProbe-1.14/1.14-1.4.37-12) it seems to have a hard dependency on a specific locally decompiled version of forge. Looking at various other mods, (JEI, HWYLA), none of them seem to have a forge version specified in their pom files.

**Comment by Ommina** [comment]
I'm using

```
    compile fg.deobf(project.dependencies.create("mcjty.theoneprobe:TheOneProbe-${top_version}") {
        transitive = false
    })
```

and it seems to work.  What it _does_, I have precious little idea.  It is a blind copy/paste from one of @McJty's other projects.  I almost hesitate to mention it here (especially on a 27-day-old issue) out of fear of spreading bad ideas, but oh well.

**Comment by yanny7** [comment]
I have same problem, also solution from @Ommina is not working for me

**Comment by pupnewfster** [comment]
I am going to close this as it seems to be fixed now with the 1.15 version `1.15-2.0.0-3`
